src/alr/alr-bootstrap.adb

```diff
@@ -27,6 +29,28 @@ package body Alr.Bootstrap is
       end if;
    end Check_Ada_Tools;
 
+   --------------------------
+   -- Checkout_Alr_Sources --
+   --------------------------
+
+   procedure Checkout_Alr_Sources (To_Path : String) is
+      Parent : constant String := Ada.Directories.Containing_Directory (To_Path);
+   begin
+      if not OS_Lib.Is_Folder (Parent) then
+         Ada.Directories.Create_Path (Parent);
+      end if;
+
+      Trace.Detail ("Checking out alr sources...");
+
+      OS_Lib.Spawn ("git", "clone " & Paths.Alr_Repo & " " & To_Path);
+
+      declare
+         Enter : OS_Lib.Folder_Guard (OS_Lib.Enter_Folder (To_Path)) with Unreferenced;
+      begin
+         OS_Lib.Spawn ("git", "submodule update --init --recursive");
+      end;
+   end Checkout_Alr_Sources;
+
```

Comment:
If bootstrap (and self-recompilation in general) is to become obsolescent, why do we do this? I’m not familiar with this part, so not sure if getting rid of self-recompilation is something that is easy to do or if it will require more work ^^

Reply:
There is still a `templates` folder used by the `init` command. At some point we can download only things really needed; right now to minimize changes I kept the checkout as it was.

Recompilation should not happen any more after #71. Commands that relied on it are broken until I finish #73 though.

Reply:
Ok, thanks for the clarification!